python/MooseDocs/testing.py

```diff
@@ -37,11 +37,28 @@ def setUpClass(cls):
     os.chdir(os.path.join(MooseDocs.MOOSE_DIR, 'docs'))
 
     config = MooseDocs.yaml_load('moosedocs.yml')
```

Comment:
This isn't something you did, but this should be MooseDocs.load_config('moosedocs.yml')